### DISCUSSION TITLE
General Root -  Service Only show one instance

### DISCUSSION THREAD

**Discussion by wzqiang1332** [apache/skywalking#12552] [Q&A]
skywalking version: 10.0.1
banyandb: 0.6.1

the first service is ok. But none of the other services can be viewed. The other services can be view in topology. log and segment for other services can be found in banyandb. How to make other services can also display normally. Other services are accessed in exactly the same way as the first service

<img width="416" alt="image" src="https://github.com/user-attachments/assets/be973b15-39b8-4663-9a77-687f123cffce">
<img width="415" alt="image" src="https://github.com/user-attachments/assets/2151fe79-b819-4140-9759-9c2687a7bd0f">
<img width="293" alt="image" src="https://github.com/user-attachments/assets/0c1b654d-c942-4dd3-953e-f87a00cd3a72">
<img width="415" alt="image" src="https://github.com/user-attachments/assets/bdb07d86-3510-4e2e-8f13-c0333d9b7b16">

**Comment by wu-sheng** [comment]
Demo on our website you are seeing is using latest BanyanDB(0.7-dev), we don't see this happens.

We have known some metadata issues and fixed it, but it was caused by low traffic services. I am not sure what is your use case.

You just put a segment stream entity here, it doesn't help. Service list is in the service_traffic measure entity.

**Reply by wzqiang1332** [reply]
It worked after switching the storage to elasticsearch. Is there a place to download banyandb-0.70-dev, or need to compile it

**Reply by wu-sheng** [reply]
Both 10.1-dev OAP and BanyanDB-0.7-dev are required at the same time, and you could download the images from their own GH repository(GH image repo)

- https://github.com/orgs/apache/packages?repo_name=skywalking
- https://github.com/orgs/apache/packages?repo_name=skywalking-banyandb
